### PR TITLE
fix: Fix references to documents.

### PR DESCRIPTION
**Pull request by numb86**
I fixed the fragment identifier of the link of warn message.

### CODE REVIEW

File: packages/test-utils/src/wrapper.js [hunk 1]
```diff
@@ -233,7 +233,7 @@ export default class Wrapper implements BaseWrapper {
     warn(
       `hasAttribute() has been deprecated and will be ` +
       `removed in version 1.0.0. Use attributes() ` +
-      `instead—https://vue-test-utils.vuejs.org/api/wrapper/#attributes`
+      `instead—https://vue-test-utils.vuejs.org/api/wrapper/#attributes-key`
```
Comment (eddyerburgh) [inline packages/test-utils/src/wrapper.js]:
```suggestion
      `instead—https://vue-test-utils.vuejs.org/api/wrapper/attributes.html`
```

File: packages/test-utils/src/wrapper.js [hunk 2]
```diff
@@ -258,7 +258,7 @@ export default class Wrapper implements BaseWrapper {
     warn(
       `hasClass() has been deprecated and will be removed ` +
       `in version 1.0.0. Use classes() ` +
-      `instead—https://vue-test-utils.vuejs.org/api/wrapper/#classes`
+      `instead—https://vue-test-utils.vuejs.org/api/wrapper/#classes-classname`
```
Comment (eddyerburgh) [inline packages/test-utils/src/wrapper.js]:
```suggestion
      `instead—https://vue-test-utils.vuejs.org/api/wrapper/classes.html`
```

File: packages/test-utils/src/wrapper.js [hunk 3]
```diff
@@ -285,7 +285,7 @@ export default class Wrapper implements BaseWrapper {
     warn(
       `hasProp() has been deprecated and will be removed ` +
       `in version 1.0.0. Use props() ` +
-      `instead—https://vue-test-utils.vuejs.org/api/wrapper/#props`
+      `instead—https://vue-test-utils.vuejs.org/api/wrapper/#props-key`
```
Comment (eddyerburgh) [inline packages/test-utils/src/wrapper.js]:
```suggestion
      `instead—https://vue-test-utils.vuejs.org/api/wrapper/props.html`
```

**Review by eddyerburgh — CHANGES_REQUESTED** [review]
Thanks, but those URLs are a problem, because they include the method arguments. Can you please use the suggested URLs instead

**Comment by numb86** [comment]
I appreciate your gently advice.
I fixed it, so please review again.

**Review by eddyerburgh — APPROVED** [review]
Thanks :)

**Pull request merged**